wyvern/events.py

```diff
@@ -111,10 +109,10 @@ def inner(listener_obj: EventListener) -> EventListener:
 
         return inner
 
-    def add_listener(self, listener: EventListener) -> None:
-        self.listeners.setdefault(listener.event_type, []).append(listener)
+    def add_listener(self, event: EventListener) -> None:
+        self.listeners.setdefault(event.event_type, []).append(event)
```

Comment:
The variable represents a EventListener, not an event, 
consider renaming the arg to `event_listener`, `lsnr` or anything that settles with an Listener.